### PR TITLE
#9 - issue templates

### LINKED ISSUE #9: issue template
Several issues, both feature and bug, do not contain enough information to get started or complete the ticket. 
A template will be added to ensure the filer supplies sufficient information.

### PR DESCRIPTION
**Pull request by kcarlson**
## Describe your changes
Issue templates to be used when submitting bugs or feature requests
## Issue ticket number and link
closes #9

### CODE REVIEW

File: ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+A clear and concise description of what the problem is. Ex. I have an issue when [...]
+
+### Describe the solution you'd like
+A clear and concise description of what you want to happen. Add any considered drawbacks.
+
+### Describe alternatives you've considered
+A clear and concise description of any alternative solutions or features you've considered.
+
+### Does this feature provide immediate value to the end customer
+Is this feature absolutely necessary? Will it benifit end users? Is this some sort of snake-oil mascarading as being the latest tech trend? Will we be coding outselves into a corner? Is this a waste of time? Are we locking ourselves into a provider, framework, service, product, subscription of which we will spend equal if not more time getting ourselves out of?
```
Comment (wooyey) [inline ISSUE_TEMPLATE/feature_request.md]: Some of these questions are rather not connected with this section, ie.:
- Are we locking ourselves into a provider, ...
- Is this a waste of time (like question "are you a terrorist?" on US and A visa form...)
How do they correspond to main question `Does this feature provide immediate value to the end customer`?
Just don't see a point to put it in here, maybe another section?

File: ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+### Possible solution
+<!--- Only if you have valid suggestions on a fix for the bug and have thorough knowledge of the code base -->
+
+### Additional information
+For example, screenshots or analysis so far.
+
+### Environment
+OS:
+Browser:
+Stack used and version:
```
Comment (wooyey) [inline ISSUE_TEMPLATE/bug_report.md:34]: I think used H/W is missing on this list...

**Review by wooyey — APPROVED** [review]
💣 👍🏻

File: ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (wooyey) [reply ISSUE_TEMPLATE/bug_report.md:34]: BTW missing versioning/tagging of docker containers in such cases...

Comment (tjonsson) [inline ISSUE_TEMPLATE/bug_report.md:34]: tag and/or branch, nightly build version

**Review by tjonsson — APPROVED** [review]

File: ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (eharhad) [reply ISSUE_TEMPLATE/feature_request.md]: Customer name if applicable.
How generic is request - how many systems is expected to order or how big money from customer we talked about.

**Review by eharhad — COMMENTED** [review]
Add an impact - Level 1,2 or 3 where:
1- Stopping BuG - need immediate action (daily)
2- Important - to be handled ASAP (week or two)
3- Minor - Can be handled (release or later)

**Review by eharhad — APPROVED** [review]

File: ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (eharhad) [reply ISSUE_TEMPLATE/bug_report.md:34]: Add an impact - Level 1,2 or 3 where:
1- Stopping BuG - need immediate action (daily)
2- Important - to be handled ASAP (week or two)
3- Minor - Can be handled (release or later)

**Review by wooyey — APPROVED** [review]

**Review by eharhad — APPROVED** [review]

**Pull request merged**